examples/basics/src/components/Welcome.astro

```diff
@@ -0,0 +1,208 @@
+				fill="#111827"></path></svg
+		>
+		<h2>What's New in Astro 5.0?</h2>
+		<p>
+			From content layers to server islands, click to learn more about the new features and
+			improvements in Astro 5.0
+		</p>
+	</a>
+</div>
+
+<style>
+	#container {
+		font-family: Inter, Roboto, 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', Arial, sans-serif;
+		font-weight: normal;
+		height: 100%;
+		width: 100%;
+	}
+
+	main {
+		width: 100%;
+		height: 100%;
+		background: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="1440" height="1024" fill="none"><g clip-path="url(%23a)"><path fill="%23FAFAFA" d="M0 0h1440v1024H0z"/><g filter="url(%23b)" opacity=".15"><path fill="url(%23c)" fill-rule="evenodd" d="M1103.43 115.43c146.42-19.45 275.33-155.84 413.5-103.59 188.09 71.13 409 212.64 407.06 413.88-1.94 201.25-259.28 278.6-414.96 405.96-130 106.35-240.24 294.39-405.6 265.3-163.7-28.8-161.93-274.12-284.34-386.66-134.95-124.06-436-101.46-445.82-284.6-9.68-180.38 247.41-246.3 413.54-316.9 101.01-42.93 207.83 21.06 316.62 6.61Z" clip-rule="evenodd"/></g><g filter="url(%23d)" opacity=".15"><path fill="url(%23e)" fill-rule="evenodd" d="M-217.58 475.75c91.82-72.02 225.52-29.38 341.2-44.74C240 415.56 372.33 315.14 466.77 384.9c102.9 76.02 44.74 246.76 90.31 366.31 29.83 78.24 90.48 136.14 129.48 210.23 57.92 109.99 169.67 208.23 155.9 331.77-13.52 121.26-103.42 264.33-224.23 281.37-141.96 20.03-232.72-220.96-374.06-196.99-151.7 25.73-172.68 330.24-325.85 315.72-128.6-12.2-110.9-230.73-128.15-358.76-12.16-90.14 65.87-176.25 44.1-264.57-26.42-107.2-167.12-163.46-176.72-273.45-10.15-116.29 33.01-248.75 124.87-320.79Z" clip-rule="evenodd"/></g></g><defs><linearGradient id="c" x1="373" x2="1995.44" y1="1100" y2="118.03" gradientUnits="userSpaceOnUse"><stop stop-color="%23D83333"/><stop offset="1" stop-color="%23F041FF"/></linearGradient><linearGradient id="e" x1="107.37" x2="1130.66" y1="1993.35" y2="1026.31" gradientUnits="userSpaceOnUse"><stop stop-color="%233245FF"/><stop offset="1" stop-color="%23BC52EE"/></linearGradient><filter id="b" width="2151" height="1700" x="73" y="-300" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/><feGaussianBlur result="effect1_foregroundBlur_674_56" stdDeviation="150"/></filter><filter id="d" width="1787.52" height="1933.31" x="-643.9" y="60.5" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/><feGaussianBlur result="effect1_foregroundBlur_674_56" stdDeviation="150"/></filter><clipPath id="a"><path fill="%23fff" d="M0 0h1440v1024H0z"/></clipPath></defs></svg>')
```

Comment:
This is super annoying, but I couldn't figure out a way to do this with CSS, not to mention with the blur and stuff the perf were.. weird. If someone has the motivation to try it out, feel free!

Reply:
I think it looking nice is also a fairly important requirement! There's a "wow effect" needed for what might be the first look someone gets of Astro

Reply:
Oh, definitely! “Simplify” doesn’t necessarily mean “make it look ugly”! It means “work with the platform to build something beautiful and performant for all users”. I think the astro.build hero section still looks nice?